src/Products/PloneMeeting/migrations/migrate_to_4218.py

```
@@ -0,0 +1,396 @@
+        load_type_from_package('MeetingConfig', 'Products.PloneMeeting:default')
+
+        self.request.set('currently_migrating_meeting_config_dx', True)
+
+        MeetingConfigMigrator.pm_migrator = self
+        walker = MeetingConfigWalker(self.portal, MeetingConfigMigrator)
+        walker()
+
+        self.request.set('currently_migrating_meeting_config_dx', False)
+        logger.info('Done.')
+
+    def run(self, extra_omitted=[], from_migration_to_4200=False):
```

Ruff (0.15.10)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)